### PR TITLE
[script.module.kodi-six] 0.1.3

### PR DESCRIPTION
**Pull request by romanvm**
### Add-on details:

- **General**
  - Add-on name: Kodi Six
  - Add-on ID: script.module.kodi-six
  - Version number: 0.1.3
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/romanvm/kodi.six
  
Wrappers around Kodi Python API that normalize handling of textual and byte strings in Python 2 and 3.

### Description of changes:

0.1.3:
- Fix non-UTF-8 byte string decoding in Python 2.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by romanvm** [comment]
This is the recent version of `kodi-six` without any changes.

File: script.module.kodi-six/addon.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.kodi-six"
+       name="Kodi Six"
+       version="0.1.3"
+       provider-name="Roman V.M.">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
```
Comment (enen92) [inline script.module.kodi-six/addon.xml]: 3.0.0?

Comment (romanvm) [reply script.module.kodi-six/addon.xml]: Oops

**Comment by enen92** [comment]
I'm wondering the reason for having kodi-six in matrix if addons there are expected to be 100% python3 compatible. Is this just a way to easy the submission of addons that are in lower branches and depend on it?

**Comment by romanvm** [comment]
It's more difficult and error-prone to maintain 2 completely different addon versions and keep them in sync regarding features and bug fixes. Having the same code for Matrix and lower version is much simpler.

This specific submission was requested here: https://github.com/xbmc/repo-plugins/pull/2725#issuecomment-593596940

**Comment by romanvm** [comment]
Travis error is a false positive.

**Comment by SylvainCecchetto** [comment]
> It's more difficult and error-prone to maintain 2 completely different addon versions and keep them in sync regarding features and bug fixes. Having the same code for Matrix and lower version is much simpler.
> 
> This specific submission was requested here: [xbmc/repo-plugins#2725 (comment)](https://github.com/xbmc/repo-plugins/pull/2725#issuecomment-593596940)

Totally agree, if kodi-six is available on Matrix (even if it does nothing) it will be easier for Python add-ons dev to maintain their code for both Leia and Matrix (just to change the `<import addon="xbmc.python" version="2.26.0"/>` line with a `sed`).

**Comment by anxdpanic** [comment]
@romanvm This still requires a version bump. 0.1.3 from leia has been installed on matrix installations up until recently, so those won't update automatically as the same version is already installed.

**Comment by enen92** [comment]
Please don't merge until travis is green, this requires a version higher than the one in leia.

**Pull request merged**